templates/data-sources/cloud_project_kube_nodepool.md.tmpl

```diff
@@ -12,8 +12,14 @@ Use this data source to get a OVHcloud Managed Kubernetes node pool.
 
 ## Example Usage
 
+* For a single-zone cluster:
+
 {{tffile "examples/data-sources/cloud_project_kube_nodepool/example_1.tf"}}
 
+* For a multi-zone cluster:
+
+{{tffile "examples/resources/cloud_project_kube_nodepool/example_4.tf"}}
```

Comment:
datasource doc pointing to an example in resources/, this seems like an error ?

Reply:
Fixed.
Remove another case.
(Too much of CTRL+F ...)